src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
```

Comment:
While browsing `redux` examples they use `containers` as name for root page components 
just like the name better than `screens` 💃 

https://github.com/reactjs/redux/tree/master/examples/real-world/src/containers